scripts/install

```diff
@@ -74,13 +78,17 @@ ynh_script_progression --message="Setting up source files..."
 
 ynh_app_setting_set --app=$app --key=final_path --value=$final_path
 # 1- Clone stable repo 
-git clone https://github.com/friendica/friendica.git -b stable "$final_path"
+ynh_exec_warn_less git clone https://github.com/friendica/friendica.git -b stable "$final_path"
```

Comment:
This will always install latest version from stable branch, which may not corespond to the version specified in `manifest.json`.

Maybe something like `git reset --hard <exact_commit_hash> would be nice to have here (the same for upgrade path with pulling from git)

Reply:
I have made some changes for that in my fork project, if you'd like to see https://github.com/YunoHost-Apps/friendica_ynh/compare/master...mr-eshua:master